### PR TITLE
Add some themes to rapps

### PR DESCRIPTION
**Pull request by petr-akhlamov**
Free WinNT 3 theme and themes from Windows.
Before install themes from Windows need accept agreement.
https://hkar.ru/121g3

### CODE REVIEW

File: longhorn5048.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[Section]
+Name = Longhorn 5048 Aero Theme
+Version = 
+License = EULA
+Description = A Windows theme from Longhorn 5048.
+Size = 455 KiB
```
Comment (learn-more) [inline longhorn5048.txt:6]: Size no longer required if SizeBytes is specified

File: longhorn5048.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+[Section]
+Name = Longhorn 5048 Aero Theme
+Version =
```
Comment (learn-more) [inline longhorn5048.txt:3]: Can just omit it instead of leave it empty?

File: luna.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+Name = Luna
+Version = 
+License = EULA
+Description = Luna theme from Windows XP.
+Size = 1.13 Mb
+Category = 15
+URLDownload = https://www.dropbox.com/s/5p87b1w28d4z0r1/Luna%20Theme.exe
+SHA1 = 584cb4bc31f6c1a252a13f925fac6e1ade049bd5
+CDPath = none
+SizeBytes = 1184890.88
```
Comment (learn-more) [inline luna.txt:11]: 0,88 bytes?
how does that work?

**Comment by petr-akhlamov** [comment]
Fixed.

**Comment by Mas4hmad** [comment]
in my computer, i have error 'check integrity failed'. picture attached.
Untitled

**Comment by learn-more** [comment]
Yeah neither of those hashes is correct.

**Comment by petr-akhlamov** [comment]
And in which program calculate hash?

**Comment by learn-more** [comment]
`sha1sum` on linux, `get-filehash` on powershell/windows, or whatever else you'd calculate a sha1 hash for a file on windows.

**Pull request merged**